src/VS.Web.CG.Msbuild/ProjectContextWriter.cs

```diff
@@ -98,91 +105,125 @@ public override bool Execute()
             msBuildContext.ProjectReferenceInformation = projReferenceInformation;
             using(var streamWriter = new StreamWriter(File.Create(OutputFile)))
             {
-                var json = JsonConvert.SerializeObject(msBuildContext);
+                var json = JsonSerializer.Serialize(msBuildContext);
                 streamWriter.Write(json);
             }
 
             return true;
         }
 
-        private IEnumerable<ProjectReferenceInformation> GetProjectDependency(
-            IEnumerable<string> projectReferences,
-            string rootProjectFullpath)
+        private IEnumerable<DependencyDescription> GetPackageDependencies(string projectAssetsFile)
         {
-            return ProjectReferenceInformationProvider.GetProjectReferenceInformation(
-                rootProjectFullpath,
-                projectReferences);
+            IList<DependencyDescription> packageDependencies = new List<DependencyDescription>();
```

Comment:
This will be replaced with the NuGet api when ready correct?

Reply:
I'm still thinking about it since it still does not give me all the information I need. I would still be dependent on project.assets.json for transitive dependencies so for now, just taking a full dependency on that file. They have not removed much from that file the nuget team says. I will add a couple more tests before shipping this so it'll be easy to react to changes as the sdk evolves. Hopefully the Nuget API catches up in a couple more releases before we go GA.